flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -550,6 +550,24 @@
               {{ checkPointConfig['checkpoints_after_tasks_finish'] ? 'Enabled' : 'Disabled' }}
             </td>
           </tr>
+          <tr
+            *ngIf="
+              checkPointConfig['state_backend'] === 'ChangelogStateBackend' &&
+              checkPointConfig['interval'] !== '0x7fffffffffffffff'
```

Comment:
Actually, even we do not set the checkpoint interval, the changelog state-backend is still enabled. Thus, I think we don't need to check the checkpoint interval.